tests/crashes/122909.rs

```diff
@@ -0,0 +1,15 @@
+//@ compile-flags: -Zpolymorphize=on -Zinline-mir=yes
+//@ known-bug: #12345
```

Comment:
```suggestion
//@ known-bug: #122909
```

Reply:
guys this is just a initial test to make sure that `compile-flags` are working :D